Makefile

```diff
@@ -46,3 +49,11 @@ install-conf: | etc/restic/b2_env.sh etc/restic/b2_pw.txt
 install-systemd:
 	install -d $(DEST_SYSTEMD)
 	install -m 0644 $(SRCS_SYSTEMD) $(DEST_SYSTEMD)
+
+			#$(RM) $$file; \
```

Comment:
Is this intended? Looked like a dead comment :upside_down_face: I'm not very sure.

Reply:
Nope, you're completely right. :D 

16e3cb2df58186cbe004b16c8a1973c702cbdc9d